guides/data/models.md

```diff
@@ -170,5 +168,5 @@ of the relationship at a time.
 
 ## Saving Data
 
-Models expose a `.save()` instance method that returns a promise which resolves
+Models expose `.save()` an instance method that returns a promise which resolves
```

Comment:
Still thinking this shouldn't be here. We're trying to say "an instance method called `.save()`", not "a method called `.save()` that saves an instance"